.github/workflows/docker-publish.yml

```diff
@@ -41,11 +41,11 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Set up JDK 17
+      - name: Set up JDK 11
```

Comment:
Why are we changing the JDK 11?

Reply:
@dubdabasoduba  The workflow file of `doocker-publish.yml` is custom written by our side. I updated it to match the JDK version defined in the `pom.xml` file using maven compiler plugin. It can be referenced [here](https://github.com/opensrp/fhir-gateway/blob/3fea9cfc4d355d7e5405e0856c0edaa39c775530/pom.xml#L77).